pkg/manager/member/tikv_upgrader.go

```diff
@@ -105,8 +105,7 @@ func (u *tikvUpgrader) Upgrade(meta metav1.Object, oldSet *apps.StatefulSet, new
 		i := podOrdinals[_i]
 		store := u.getStoreByOrdinal(meta.GetName(), *status, i)
 		if store == nil {
-			setUpgradePartition(newSet, i)
```

Comment:
Why remove this? If there are 3 TiKVs, tikv-2 is up but tikv-0 and tikv-1 are Pending, in this case, the tikv-0 and tikv-1 cannot be upgraded because there is no store for them.